integration/setup/common.go

```diff
@@ -93,11 +98,65 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 	}
 }
 
+// buildMongoDBURIOpts represents buildMongoDBURI's options.
+type buildMongoDBURIOpts struct {
```

Comment:
I see in the `buildMongoDBURI` there is the logic that only one of `hostPort` or `unixSocketPath` needs to be set. Should we add any comments about it here? Like "it's up to X to decide how to use this configuration".

Reply:
The logic that ensures that only one parameter should be set in `buildMongoDBURI` will fail test execution, so I think it's ok to leave it as is.